### PR TITLE
ouch: add page

### PR DESCRIPTION
**Pull request by siph**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** 0.3.1

### CODE REVIEW

File: pages/common/ouch.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# ouch
+
+> Painless compression and decompression for your terminal.
+> More information: <https://crates.io/crates/ouch>.
+
+- Decompress file:
+
+`ouch decompress {{archive.tar.xz}}`
```
Comment (blueskyson) [inline pages/common/ouch.md:8]:
```suggestion
`ouch decompress {{path/to/archive.tar.xz}}`
```

File: pages/common/ouch.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+- Decompress file to specific location:
+
+`ouch decompress {{archive.tar.xz}} --dir {{path/to/directory}}`
```
Comment (blueskyson) [inline pages/common/ouch.md:12]:
```suggestion
`ouch decompress {{path/to/archive.tar.xz}} --dir {{path/to/directory}}`
```

File: pages/common/ouch.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+- Decompress multiple files:
+
+`ouch decompress {{archive1.tar archive2.tar.gz ...}}`
```
Comment (blueskyson) [inline pages/common/ouch.md:16]:
```suggestion
`ouch decompress {{path/to/archive1.tar path/to/archive2.tar.gz ...}}`
```

File: pages/common/ouch.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+- Compress files:
+
+`ouch compress {{index.js index.html ...}} {{archive.zip}}`
```
Comment (blueskyson) [inline pages/common/ouch.md:20]:
```suggestion
`ouch compress {{path/to/index.js path/to/index.html ...}} {{path/to/archive.zip}}`
```

**Review by blueskyson — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages/common/ouch.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+# ouch
+
+> Painless compression and decompression for your terminal.
```
Comment (navarroaxel) [inline pages/common/ouch.md:3]: We prefer an objective description
```suggestion
> Command-line utility for compressing and decompressing files and directories.
```

File: pages/common/ouch.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+> More information: <https://crates.io/crates/ouch>.
+
+- Decompress file:
```
Comment (navarroaxel) [inline pages/common/ouch.md:6]:
```suggestion
- Decompress a specific file:
```

File: pages/common/ouch.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+
+`ouch decompress {{path/to/archive.tar.xz}}`
+
+- Decompress file to specific location:
```
Comment (navarroaxel) [inline pages/common/ouch.md:10]:
```suggestion
- Decompress a file to a specific location:
```

File: pages/common/ouch.md [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+
+`ouch decompress {{path/to/archive.tar.xz}} --dir {{path/to/directory}}`
+
+- Decompress multiple files:
+
+`ouch decompress {{path/to/archive1.tar path/to/archive2.tar.gz ...}}`
+
+- Compress files:
+
+`ouch compress {{path/to/index.js path/to/index.html ...}} {{path/to/archive.zip}}`
```
Comment (navarroaxel) [inline pages/common/ouch.md:20]: This is more standard
```suggestion
`ouch compress {{path/to/file1 path/to/file2 ...}} {{path/to/archive.zip}}`
```

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**